tests/mypy/mypy_function_wrapper_fn_t2.py:20: error: Argument 1 to "function_wrapper" has incompatible type "Callable[[str, Any, tuple[Any, ...], dict[str, Any]], Any]"; expected "Callable[[Callable[[VarArg(Never), KwArg(Never)], Never], Any, tuple[Any, ...], dict[str, Any]], Any]"  [arg-type]
tests/mypy/mypy_function_wrapper_fn_t2.py:27: error: Need type annotation for "wrapped_function"  [var-annotated]
tests/mypy/mypy_function_wrapper_fn_t2.py:27: error: Argument 1 to "__call__" of "FunctionDecorator" has incompatible type "Callable[[int, str], str]"; expected "Callable[[VarArg(Never), KwArg(Never)], Never]"  [arg-type]
Found 3 errors in 1 file (checked 1 source file)
